_data/snippets.yml

```diff
@@ -114,3 +114,31 @@ enterprise_3_encryption_key_backup: |
   1. Make sure you have appropriate access to the kubernetes cluster: you need credentials for `kubectl` and connection to [travis-api-pod] 
   2. Run `kubectl exec -it [travis-api-pod] cat /app/config/travis.yml |grep -A 2 encryption` (using Travis API pod is recommended). 
   3. Create a backup of the value returned by that command by either writing it down on a piece of paper or storing it on a different computer.
+github_oauth_access_rights: | 
+  When you sign in to [travis-ci.com](https://travis-ci.com) using GitHub for the first time, you will receive a message from GitHub saying:
+  
+  > Travis CI by travis-pro wants to access your [account name] account.
+  
+  and in the repositorie's section it will state: 
+  
+  > This application will be able to read and write all public and private repository data.
+  
+  This is not how Travis CI access yout data, however we can explain it later in the process. 
+  
+  The warning is triggerred due to GitHub OAuth App permissions which Travis CI uses, and due to the available granularity of permission scopes (see GitHub [Scopes for OAuth Apps documentation](https://docs.github.com/en/developers/apps/scopes-for-oauth-apps) ).
+  
+  **Travis CI does not access all your repositories by default**. 
+  
+  Once you acknowledge the access rights, you will see Travis CI OAuth application present in your GitHub [Authorized OAuth Apps list](https://github.com/settings/applications), however **you need to explicitly configure which repositories Travis CI has access to** within your [travis-ci.com](https://travis-ci.com) account. The configuration is done during the activation of Travis CI for your repositories. You can use either the 'All repositories' option or the 'Only select repositories' option during the activation process.
+  
+  When the Travis CI activation has completed, you will see the actual Travis CI GitHub Application installed in [Installed GitHub Apps](https://github.com/settings/installations) section.
```

Comment:
When the Travis CI activation has completed, you will see the actual Travis CI GitHub Application installed in **the** [Installed GitHub Apps](https://github.com/settings/installations) section.